pkg/controller/account/iam.go

```diff
@@ -388,6 +395,71 @@ func deleteAllAccessKeys(reqLogger logr.Logger, client awsclient.Client, userNam
 	return nil
 }
 
+// CreateAccount creates an AWS account for the specified accountName and accountEmail in the orgnization
+func CreateAccount(reqLogger logr.Logger, client awsclient.Client, accountName, accountEmail string) (*organizations.DescribeCreateAccountStatusOutput, error) {
```

Comment:
Is this move from another PR?

Reply:
Yeah.  It's one of the reasons I made this a WIP because I still have to clean this up, but I wanted to get this up so I remembered what I was doing on Friday when Monday came 😂